src/docs/asciidoc/index.adoc

```diff
@@ -72,46 +72,47 @@ Run your build and pass the parameters to it using `-P` commandline parameters:
 
 == Setup
 
-In order to use this plugin you need to set up an included build that will produce the resulting plugin.
+In order to use this plugin you need an included build that will produce the resulting plugin.
 A typical project layout looks like this:
 
 ```
 my-project
- |- build-logic
- |   |- build-parameters
- |   |  |- build.gradle.kts <3>
- |   |- settings.gradle.kts <2>
+ |- gradle
+ |   |- plugins
```

Comment:
`gradle/plugins` is now the official recommendation via `gradle init` (see: https://github.com/gradle/gradle/pull/21192)